backend/src/map_model.rs

```diff
@@ -46,6 +46,11 @@ pub struct MapModel {
     pub modal_filters: BTreeMap<RoadID, ModalFilter>,
     pub diagonal_filters: BTreeMap<IntersectionID, DiagonalFilter>,
 
+    /// Indexed by IntersectionID. For each intersection, a list of (from, to) roads that are not
```

Comment:
Mirroring the structures above for editable things, rather than bundling with each intersection. This is expected to be a pretty sparse list -- most intersections don't have TRs.